build_runner/lib/src/generate/build_definition.dart

```diff
@@ -60,7 +60,12 @@ class _Loader {
     if (!_options.writeToCache &&
         _buildActions.any((action) =>
```

Comment:
I think you can just remove this check now right? (the same check effectively happens inside this if now)

Reply:
Done.